newrelic/structures_newrelic_nrql_alert_condition.go

```diff
@@ -171,6 +171,13 @@ func expandNrqlConditionTerm(term map[string]interface{}, conditionType, priorit
 		return nil, fmt.Errorf("one of `duration` or `threshold_duration` must be configured for block `term`, but not both")
 	}
 
+	switch conditionType {
+	case "baseline", "outlier":
+		if strings.ToUpper(term["operator"].(string)) != "ABOVE" {
```

Comment:
I think we have these as an enum value in the client

Reply:
Updated.